src/main/java/at/hannibal2/skyhanni/features/fishing/trophy/GeyserOptions.kt

```diff
@@ -0,0 +1,72 @@
+package at.hannibal2.skyhanni.features.fishing.trophy
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.FishingBobberCastEvent
+import at.hannibal2.skyhanni.events.LorenzRenderWorldEvent
+import at.hannibal2.skyhanni.events.ReceiveParticleEvent
+import at.hannibal2.skyhanni.utils.LocationUtils.distanceTo
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RenderUtils.drawFilledBoundingBox_nea
+import at.hannibal2.skyhanni.utils.SpecialColour
+import net.minecraft.entity.projectile.EntityFishHook
+import net.minecraft.util.AxisAlignedBB
+import net.minecraft.util.EnumParticleTypes
+import net.minecraftforge.fml.common.eventhandler.EventPriority
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.awt.Color
+
+class GeyserOptions {
+    private val config get() = SkyHanniMod.feature.fishing.trophyFishing.geyserOptions
+
+    private var bobber: EntityFishHook? = null
+    private var geyser: LorenzVec? = null
+
+    @SubscribeEvent(priority = EventPriority.LOW, receiveCanceled = true)
+    fun onPacket(event: ReceiveParticleEvent) {
```

Comment:
```suggestion
    fun onReceiveParticle(event: ReceiveParticleEvent) {
```
I know some other parts of the code use packet or even chat but I believe these are older or copy pastes that were not caught, this fits with most of the events throughout the mod

Reply:
yep your correct i had tried to use packets before realising i was using wrong event